dwt_util.py

```diff
@@ -65,9 +65,8 @@ def stdout(self, value):
 
 def is_64bit():
     if os.name == 'nt':
-        output = subprocess.check_output(['wmic', 'os', 'get', 'OSArchitecture'])
-        os_arch = output.split()[1]
-        return True if os_arch == '64-bit' else False
+        os_arch = os.environ["PROCESSOR_ARCHITEW6432"]
```

Comment:
It seems that `PROCESSOR_ARCHITECTURE` environment variable is better suited for the task.

`PROCESSOR_ARCHITECTURE` gives the actual processor architecture: "x86", "AMD64" or "IA64".

`PROCESSOR_ARCHITEW6432` gives the same value as PROCESSOR_ARCHITECTURE, but it is only set on 32-bit processes. So it can be used to to detect whether the process is in a WoW64 environment or not.

At least that's what I can gather from (MSDN)[https://docs.microsoft.com/en-us/windows/win32/winprog64/wow64-implementation-details#environment-variables]

Reply:
Good catch! That would've been a nasty thing to slip up on. Should have a fix ready shortly.